dubbo-container/dubbo-container-api/src/main/java/org/apache/dubbo/container/Main.java

```diff
@@ -61,9 +61,10 @@ public static void main(String[] args) {
             final List<Container> containers = new ArrayList<Container>();
             for (int i = 0; i < args.length; i++) {
                 Container container = LOADER.getExtension(args[i]);
-                if (null != container) {
-                    containers.add(LOADER.getExtension(args[i]));
+                if (null == container) {
+                    throw new IllegalArgumentException("Default extension is not configured, but the extension name is 'true'");

```

Comment:
msg can be String.format("The extension is not find, pls check your param: %s is match spi", args[i]);